compliance/collection/nodeinventorizer/nodeinventory_test.go

```diff
@@ -0,0 +1,211 @@
+package nodeinventorizer
+
+import (
+	"testing"
+
+	"github.com/stackrox/scanner/database"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestNodeInventorizer(t *testing.T) {
+	suite.Run(t, new(NodeInventorizerTestSuite))
+}
+
+type NodeInventorizerTestSuite struct {
+	suite.Suite
+}
+
+func (s *NodeInventorizerTestSuite) TestConvertRHELComponentIDs() {
+	testCases := map[string]struct {
+		inComponents  []*database.RHELv2Package
+		outComponents []*scannerV1.RHELComponent
```

Comment:
Looks like this is leftover from a previous state of this test, where you compared in against out. I think it can be removed now.

Reply:
Thank you for spotting this. Both, this and the `convertedIds` were leftovers of earlier iterations